dashboards/ec2/ec2_keys_detail.sp

```diff
@@ -1,6 +1,6 @@
 dashboard "aws_ec2_keys_relationships" {
```

Comment:
Can you please rename this file to `ec2_key_pair_detail.sp`? All files use a singular resource and the full resource type should be in the name.